xtask/Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name = "xtask"
+version = "0.1.0"
+authors.workspace = true
+edition.workspace = true
+
+[dependencies]
+fastn-xtask = { path = "/Users/ayushdeolasee/Developer/fastn/fastn/fastn-xtask" }
+clap = { version = "4.0", features = ["derive"] }
+tokio = { version = "1.36", features = ["full"] }
```

Comment:
Unless there is a strong reason pick the least precise dependencies. Eg do we not work against Tokio 1.0? Why are we specifically asking for 1.36? Did 1.36 have something that we need but 1.35 did not? Then it's fair to say we need 1.36, else just say we need 1.